src/components/Tabs/Tabs.tsx

```diff
@@ -30,21 +28,21 @@ const Trigger = styled(RadixTabs.Trigger)`
   }
 
   &:hover {
-    border-bottom: 2px solid ${props => props.theme.click.tabs.basic.stroke.hover};
-    background-color: ${props => props.theme.click.tabs.basic.color.background.hover};
```

Comment:
background is removed

Reply:
What was the background colour got moved into the border-bottom attribute, so the background is still present.

Reply:
Sorry, just re-read your comment, fixed.